CodeKata/MonkeyCountArray.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CodeKata
+{
+    public class MonkeyCountArray
+    {
+        public static int[] MonkeyCount(int n)
```

Comment:
You can do the same with Range method: 
```return Enumerable.Range(1, n).ToArray();```

Reply:
I know, I saw it among solutions after completing task